patches/chrome-browser-resources-settings-privacy_page-privacy_page.js.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.js b/chrome/browser/resources/settings/privacy_page/privacy_page.js
+index c8a41ea859437ec0c494f05bdbc15ee5fd5423ec..5def6736ec607bd367ecc650f128786b5e5314e5 100644
+--- a/chrome/browser/resources/settings/privacy_page/privacy_page.js
++++ b/chrome/browser/resources/settings/privacy_page/privacy_page.js
+@@ -52,7 +52,7 @@ Polymer({
+       }
+     },
+ 
+-    // <if expr="_google_chrome and not chromeos">
++    // <if expr="_chromium and not chromeos">
```

Comment:
Is this changing for documenting? (IMO, we don't need to change comment that create patch files)

Reply:
It actually seemed to be some kind of pre-processing templating system used by the JavaScript files, but "hidden" as comments.

Reply:
Oh.. I didn't know about it. Thanks for pointing out!